prime-router/metadata/fhir_mapping/hl7/message/ORM_O01.yml

```diff
@@ -29,7 +29,7 @@ resources:
   - resourceName: Observation
     segment: .ORDER_DETAIL.OBSERVATION.OBX
     group: ORDER
-    resourcePath: resource/Observation
+    resourcePath: segments/OBX/Observation
```

Comment:
good call, looks like ServiceRequest might need to get updated to while you're here?